src/php/types/object.rs

```diff
@@ -161,6 +164,38 @@ impl ZendObject {
     pub(crate) fn refcount_inc(&mut self) {
         self.gc.refcount += 1;
     }
+
+    pub(crate) fn to_string(&self) -> String {
```

Comment:
Should probably change function name eventually, although I'm not sure what to...

Reply:
`strval`?

Reply:
Could do `as_string`, since `to_` usually implies consuming `self`.

Reply:
Aren't `as_` functions supposed to be cheap though? This invokes arbitrary PHP.

Reply:
Also, for what its worth, `to_` should be fine idiomatically for this: https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv

Consider the `str::to_lowercase` as an example.

Reply:
Yeah true, `strval` sounds good then?